File: Programming Language(ver.2)/Go/109.go
package main
import "fmt"

func main(){
	var score int
	fmt.Scanln(&score)
	if score > 100 || score < 0 {
		fmt.Println("error")
		return
	}
	if score >= 60 {
		fmt.Println("pass")
	}else{
		fmt.Println("fail")
	}
	if score % 2 == 0 {
		fmt.Println("even")
	}else{
		fmt.Println("odd")
	}
}

File: Programming Language(ver.2)/C++/109.cpp
#include <iostream>
using namespace std;

int main()
{
    int score;
    cin >> score;
    if (score < 0 || score > 100) {
        cout << "error" << endl;
    }
    else if (score >= 60) {
        cout << "pass" << endl;
        if (score % 2 == 1)
            cout << "odd";
        else
            cout << "even";
    }
    else {
        cout << "fail" << endl;
        if (score % 2 == 1)
            cout << "odd";
        else
            cout << "even";
    }
    return (0);
}